src/foam/u2/detail/TabbedDetailView.js

```diff
@@ -73,13 +73,16 @@ foam.CLASS({
                         code: s.title
                       }) :
                       s.title$;
+
+                    var tab = foam.core.SimpleSlot.create();
                     this
-                      .start(self.Tab, { label$: title$ || self.defaultSectionLabel })
+                      .start(self.Tab, { label$: title$ || self.defaultSectionLabel }, tab)
                         .call(function() {
                           this.tag(self.SectionView, {
                             data$: self.data$,
                             section: s,
-                            showTitle: false
+                            showTitle: false,
+                            skipLoading$: tab.value.selected$.map(function(s) {return !s;})
```

Comment:
Space after !.
Does this then erase the tab from the UI when it is unselected, forcing us to rebuild if the user goes back?
If so, it would be better to make skipLoading a latch and rename it loadLatch. This could be done by adding a preSet so that it can toggle from false to true but not the reverse (making it a latch).

Reply:
space fixed.
got a small problem though:
skipLoading is reference which is shared between all the tabs, so if will do 
`
preSet: function(o, n) {
        if ( o && ! n ) {
          return o;
        }
        return n;
      }
`
then tabs after first will not be updated

Reply:
I got your point though. 
I'll try to fix the code but I have no idea how to do that with the approach mentioned above